### PR TITLE
json save/load option

### PR DESCRIPTION
**Pull request by Rewalon**

### CODE REVIEW

File: edf/FindForm.cs [hunk 1]
```diff
@@ -4,15 +4,24 @@
...
 
 namespace SubReal.EasyDuplicatesFinder
 {
     public partial class FindForm : Form
     {
-        private EdfFiles _edfFiles;
+        private EdfFiles _edfFiles;     
+        
+        const string optionsFileName = "optionsefd.json";
+        public class OptionsEDF
```
Comment (AlexSnowLeo) [inline edf/FindForm.cs:17]: @Rewalon, Why are you using a nested class?

File: edf/FindForm.cs [hunk 2]
```diff
@@ -4,15 +4,24 @@
...
 
 
 namespace SubReal.EasyDuplicatesFinder
 {
     public partial class FindForm : Form
     {
-        private EdfFiles _edfFiles;
+        private EdfFiles _edfFiles;     
+        
+        const string optionsFileName = "optionsefd.json";
```
Comment (AlexSnowLeo) [inline edf/FindForm.cs:16]: Typo in file name? I think we could use 'edf-settings.json' instead.

File: edf/FindForm.cs [hunk 3]
```diff
@@ -4,15 +4,24 @@
 using System.IO;
 using System.Reflection;
 using System.Windows.Forms;
+using System.Text.Json;
```
Comment (AlexSnowLeo) [inline edf/FindForm.cs:7]: Could you try to migrate to the NewtonsoftJSON library, this is the preferred choice in our case.

Comment (AlexSnowLeo) [reply edf/FindForm.cs:7]: Discussed. Let's leave this library.

File: edf/FindForm.cs (hunk 1 above)
Comment (Rewalon) [reply edf/FindForm.cs:17]: it is not used anywhere else

File: edf/FindForm.cs (hunk 2 above)
Comment (Rewalon) [reply edf/FindForm.cs:16]: yes is it. I agree with the suggested name.

File: edf/FindForm.cs (hunk 3 above)
Comment (Rewalon) [reply edf/FindForm.cs:7]: Native solution.
Also - https://samiprogramming.medium.com/system-text-json-vs-newtonsoft-json-d01935068143

**Pull request merged**